taxcalc/tests/test_functions.py

```diff
@@ -0,0 +1,40 @@
+"""
+Tests for Tax-Calculator functions.py file.
+"""
+# CODING-STYLE CHECKS:
+# pep8 --ignore=E402 test_functions.py
+# pylint --disable=locally-disabled test_functions.py
+# (when importing numpy, add "--extension-pkg-whitelist=numpy" pylint option)
+
+import os
+import sys
+import re
+CUR_PATH = os.path.abspath(os.path.dirname(__file__))
+
+
+def test_function_args_usage():
+    """
+    Checks each function argument in functions.py for use in its function body.
+    """
+    funcfilename = '{}/../functions.py'.format(CUR_PATH)
```

Comment:
@talumbau said:
> something like ```os.path.join(CUR_PATH, '..', 'functions.py')``` is probably best and is cross-platform (handles \ for path separator for Windows platform).

OK, I followed your suggestion, but you should note that most of the other test_*.py files violate your recommendation.   For example, in test_calculate.py, we have the following code:
```
TAX_DTA_PATH = os.path.join(CUR_PATH, '../../tax_all1991_puf.gz')
```
Are you saying that test_calculate.py will fail (because of the forward slashes) when run under Windows?  Is anybody on the Tax-Calculator development team working under Windows?

cc @MattHJensen @Amy-Xu @GoFroggyRun